test/Microsoft.AspNet.Localization.Tests/AccessLanguageHeaderRequestCultureProviderTest.cs

```diff
@@ -106,5 +106,38 @@ public class AccessLanguageHeaderRequestCultureProviderTest
                 var response = await client.GetAsync(string.Empty);
             }
         }
+        
+        [Fact]
+        public async void OmitDefaultRequestCultureShouldNotThrowNullReferenceException_And_ShouldGetTheRightCulture()
```

Comment:
@kirthik the main difference between OmitDefaultRequestCultureShouldNotThrowNullReferenceException_And_ShouldGetTheRightCulture and GetFallbackLanguage_ReturnsFirstNonNullCultureFromSupportedCultureList is https://github.com/aspnet/Localization/blob/dev/test/Microsoft.AspNet.Localization.Tests/AccessLanguageHeaderRequestCultureProviderTest.cs#L55